src/styles/sections/sections.less

```diff
@@ -26,3 +28,122 @@ img {
 .centered {
   text-align: center;
 }
+
+section {
+  @media (max-width: @mobile-width) {
+    top: 0;
+  }
+
+  position: relative;
+  top: @top-offset;
+}
+
+section.hero {
+  text-align: center;
+  display: flex;
+  height: calc(100vh - 100px);
+  align-items: center;
+  justify-content: center;
+  top: 0;
+}
+
+section.about {
+  @media (max-width: @mobile-width) {
+    flex-direction: column;
+    height: auto;
+  }
+
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  height: calc(100vh - 100px);
+
+  .item {
+    @media (max-width: @mobile-width) {
+      width: 100%;
+      background-position: top center;
+      background-size: 250px 250px;
+      padding-top: 250px;
+      padding-right: 0px;
+    }
+
+    width: calc(33% - 32px);
+    background-position: top left;
+    background-size: 350px 350px;
+    padding-top: 350px;
+    padding-right: 32px;
+    background-repeat: no-repeat;
+    background-image: url('/assets/images/drawings/dandelions_Page_1 1 (Traced)-5.svg');
+  }
+
+  .item:first-child {
+    background-image: url('/assets/images/drawings/dandelions_Page_2 1 (Traced)-4.svg');
+  }
+
+  .item:last-child {
+    @media (max-width: @mobile-width) {
+      width: 100%;
+    }
+
+    padding-right: 0;
+    width: 33%;
+    background-image: url('/assets/images/drawings/dandelions_Page_2 1 (Traced)-3.svg');
+  }
+}
+
+section.grantmakers,
+section.artists,
+section.directory,
+section.connect {
+  @media (max-width: @mobile-width) {
+    top: 0;
+    margin: 25vh 0;
+  }
+
+  text-align: center;
+  position: relative;
+  top: @top-offset;
+  margin: 0 0 25vh;
+
+  h2 {
+    margin: 0;
+  }
+}
+
+section.report,
+section.deck,
+section.podcast,
+section.courses,
+section.examples,
+section.contact {
+  max-width: 720px;
+  margin: 0 auto;
+}
+
+section.deck,
+section.courses,
+section.examples,
+section.igfeed {
+  @media (max-width: @mobile-width) {
+    margin-bottom: 0;
+  }
+
+  margin-bottom: 25vh;
+}
+
+section.contact {
+  @media (max-width: @mobile-width) {
+    margin-bottom: 0;
+  }
+
+  margin-bottom: calc(25vh - 128px);
+}
+
+section.footer {
+  font-size: @font-size-sm;
+  line-height: 20px;
+  color: gray;
+  max-width: 720px;
+  margin: 0 auto;
+  top: 0;
+}
```

Comment:
Styling for specific sections should be in their own files, like the other sections. Otherwise this file will grow unbounded.

Reply:
Thank you for this. I moved specific styles to separate files and used a common class for sections that display as titles. Additional classes that should be applied to the <section> element itself are assigned via the md metadata `classes` property.